aws/resource_aws_elasticache_cluster.go

```diff
@@ -438,7 +438,7 @@ func resourceAwsElasticacheClusterRead(d *schema.ResourceData, meta interface{})
 		if c.ConfigurationEndpoint != nil {
 			d.Set("port", c.ConfigurationEndpoint.Port)
 			d.Set("configuration_endpoint", aws.String(fmt.Sprintf("%s:%d", *c.ConfigurationEndpoint.Address, *c.ConfigurationEndpoint.Port)))
-			d.Set("cluster_address", aws.String(fmt.Sprintf("%s", *c.ConfigurationEndpoint.Address)))
+			d.Set("cluster_address", aws.String((*c.ConfigurationEndpoint.Address)))
```

Comment:
Nit: Its easier to just passthrough the existing string pointer instead of dereferencing it and then creating a pointer again. `d.Set("cluster_address", c.ConfigurationEndpoint.Address)`